front/app/containers/IdeasNewPage/IdeasNewSurveyForm/SurveyHeading.tsx

```diff
@@ -64,148 +52,103 @@ const SurveyHeading = ({
     setShowLeaveModal(false);
   };
 
-  return modalPortalElement
-    ? createPortal(
-        <Box
-          display="flex"
-          flexDirection="row"
-          justifyContent="center"
-          w="100%"
-          zIndex="1010"
-          position="fixed"
-          borderRadius="2px"
+  if (!project) return null;
+
+  const showEditSurveyButton =
+    !isSmallerThanPhone && canModerateProject(project.data.id, authUser);
+  const phaseId =
+    searchParams.get('phase_id') ||
+    project.data.relationships.current_phase?.data?.id;
+  const linkToSurveyBuilder: RouteType = `/admin/projects/${project.data.id}/phases/${phaseId}/native-survey/edit`;
+
+  return (
+    <>
+      <Box
```

Comment:
This file again had loads of needless Box wrapping and CSS styles. We should be much more careful with these. They cloud our code, and we become afraid to make changes and break things.